### ISSUE TITLE
The loss curve is normal？

### ISSUE THREAD

**Issue by Passenger12138** [cyq373/SSD-GAN#5]
I am now training on the celebA dataset. I am not sure whether the training results are not good because of the unreasonable settings of the training hyperparameters or the insufficient number of iterations. I would like to know the correct loss curve style to help me further understand.
